packages/rainbow-sprinkles/README.md

```diff
@@ -99,7 +100,7 @@ Then set-up in your "host" component (in this case, a Box component):
 
 ```typescript
 // Box.css.ts
-import { createRainbowSprinklesCss } from './rainbow-sprinkles';
+import { createRainbowSprinklesCss } from './rainbow-sprinkles.css';
```

Comment:
The rainbow-sprinkles file should not be a `.css.js` file, as demonstrated here https://github.com/wayfair-incubator/rainbow-sprinkles/blob/main/example/rainbow-sprinkles.ts

Otherwise Vanilla Extract will throw an error on build

Reply:
Ah ok yea I think the code snippet at the top says it's in a css ts file, I'll update the pr with that change

Reply:
Updated 👍